extensions/azure/azure-test/src/testFixtures/java/org/eclipse/dataspaceconnector/azure/testfixtures/annotations/AzureStorageIntegrationTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (c) 2020, 2021 Microsoft Corporation
```

Comment:
nit: same for the dates
```suggestion
 *  Copyright (c) 2020, 2021, 2022 Microsoft Corporation
```